sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2265,6 +2265,33 @@ object GpuOverrides extends Logging {
         // Last does not overflow, so it doesn't need the ANSI check
         override val needsAnsiCheck: Boolean = false
       }),
+    expr[MinBy](
```

Comment:
Does it support Float as the order type, especially when the column has NaNs? Spark-rapids has a different way to deal with float type in GpuMin.

Reply:
@thirtiseven Please take care of this in cuDF change

Reply:
related issue: https://github.com/NVIDIA/spark-rapids/issues/5320